components/Currency/CurrencyIcon/TokenIcon.tsx

```diff
@@ -16,11 +18,15 @@ export type TokenIconProps = {
 };
 
 const TokenIcon: React.FC<TokenIconProps> = ({ currencyKey, ...props }) => {
-	const OneInchTokenListQuery = useOneInchTokenList();
-	const OneInchTokenListMap = OneInchTokenListQuery.data?.tokensMap ?? null;
+	// TODO: Find a way to do this without checking if we're on L2 or not.
+	const isL2 = useIsL2();
+	const tokenMap: any = React.useMemo(
```

Comment:
I can change this, but I personally prefer to do this because it makes it easier to reach for hooks. Instead of having to add a new import, I can just do `React.useCallback` or `React.FC` and it just works. Is there a specific reason you prefer the other one, or is this just for consensus/convention?